src/Sulu/Bundle/ContentBundle/Resources/views/Template/macros/properties.html.twig

```diff
@@ -0,0 +1,75 @@
+<div class="grid-row">
+{# initialize columns #}
+{% set columns = 0 %}
+{% for property in properties %}
+
+    {# set propertyColspan with default if colspan is not set #}
+    {% set propertyColspan = property.colspan != "" ? property.colspan : 12 %}
+
+    {# create new row if there are more then 12 colspans in a row #}
+    {% if columns + propertyColspan <= 12 %}
+        {# set current amount of colspans #}
+        {% set columns = columns + propertyColspan %}
+    {% else %}
+        </div>
+        <div class="grid-row">
+        {# set current amount of colspans #}
+        {% set columns = propertyColspan %}
+    {% endif %}
+
+    {# get params for property #}
+    {% set params = sulu_get_params(property) %}
+    {% set id = propertyPrefix|default('') ~ property.name ~ propertyPostfix|default('') %}
+
+    {% if property.contentTypeName == 'section' %}
+        {% include 'SuluContentBundle:Template:macros/section.html.twig' with {
+        'property': property,
```

Comment:
this format is quite hard to read shouldnt we intend this parameters with 4 spaces? i know intellij will remove the always but for readability i would do it.